DEVELOPER_GUIDE.md

```diff
@@ -6,7 +6,8 @@
 	- [Git Clone OpenSearch Repo](#git-clone-opensearch-repo)
 	- [Publish OpenSearch Feature/Extensions branch to Maven local](#publish-opensearch-feature/extensions-branch-to-maven-local)
 	- [Run OpenSearch-SDK](#run-opensearch-sdk)
-	- [Run OpenSearch](#run-opensearch)
+	    - [Create extensions.yml file](#create_extensions_file)
+        - [Run OpenSearch](#run-opensearch)
```

Comment:
In the markdown preview this "Run OpenSearch" is indented along with the line above. Not sure if that's a bug in the previewer or some whitespace issue or if you need a space.  But not sure either one should be indented.